packages/shared-ux/button_toolbar/src/buttons/toolbar_button/toolbar_button.styles.ts

```diff
@@ -6,15 +6,14 @@
  * Side Public License, v 1.
  */
 
-// Temporary fix for lensApp icon not support ghost color
-.solutionToolbar__primaryButton--dark {
-  .euiIcon path {
-    fill: $euiColorInk;
-  }
-}
+import { UseEuiTheme } from '@elastic/eui';
 
-.solutionToolbar__primaryButton--light {
-  .euiIcon path {
-    fill: $euiColorGhost;
-  }
-}
+export const ToolbarButtonStyles = ({ euiTheme }: UseEuiTheme) => {
+  return {
+    emptyButton: {
+      backgroundColor: euiTheme.colors.emptyShade,
+      border: `${euiTheme.border.thin} !important`,
+      color: `${euiTheme.colors.text}`,
```

Comment:
This is the styling to create the following type of button: 

![image](https://user-images.githubusercontent.com/8698078/220714167-cddd10f7-8a05-43e9-802c-93564645fd91.png)